HelpSource/Tutorials/QT/compose_complex_view.schelp

```diff
@@ -0,0 +1,395 @@
+.drawLines_(true)
+.elasticMode_(1);
+
+multiSliderHideButton = Button()
+.string_("Hide MultiSlider")
+.maxHeight_(64);
+
+// Show button is not visible on startup
+multiSliderShowButton = Button()
+.visible_(false)
+.string_("Show MultiSlider")
+.maxHeight_(64);
+
+multiSliderView.layout.add(
+	multiSliderHideButton, 1);
+multiSliderView.layout.add(
+	multiSlider, 9);
+
+// Insert the subview into the main view
+mainView.layout.add(multiSliderView, 1);
+// Add the 'show' button to the visibility view
+visibilityView.layout.add(multiSliderShowButton);
+
+// Setup the text view
+textView = UserView()
+.layout_(VLayout());
+
+text = StaticText()
+.string_("Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor. Cras elementum ultrices diam.")
+.align_(\center);
+
+textHideButton = Button()
+.string_("Hide Text")
+.maxHeight_(64);
+
+// Show button is not visible on startup
+textShowButton = Button()
+.visible_(false)
+.string_("Show Text")
+.maxHeight_(64);
+
+textView.layout.add(
+	textHideButton, 1);
+textView.layout.add(
+	text, 9);
+
+// Insert the subview into the main view
+mainView.layout.add(textView, 1);
+// Add the 'show' button to the visibility view
+visibilityView.layout.add(textShowButton);
+
+// Agencing the top level
+window.layout_(
+	VLayout()
+	.add(mainView)
+	.add(visibilityView)
+);
+
+// Hide the visibility view,
+// because everything is visible at start
+visibilityView.visible_(false);
+
+
+// --- Here's the visibilty logic : ---
+// When the hide button is pressed
+multiSliderHideButton.action_({
+	// Hide the view
+	multiSliderView.visible_(false);
+	// Set the show button visible
+	multiSliderShowButton.visible_(true);
+	// Increment the number of hidden views
+	numberOfHiddenView = numberOfHiddenView + 1;
+	// If exactly one item is hidden,
+	// this means we have to show the visibility view
+	// which was previously hidden
+	if(numberOfHiddenView == 1)
+	{ visibilityView.visible_(true); };
+});
+
+// Now the inverse for the show button
+// When the show button is pressed
+multiSliderShowButton.action_({
+	// Show the view
+	multiSliderView.visible_(true);
+	// Hide the show button
+	multiSliderShowButton.visible_(false);
+	// Decrement the number of hidden views
+	numberOfHiddenView = numberOfHiddenView - 1;
+	// If there is no hidden item,
+	// hide the visibility view
+	if(numberOfHiddenView == 0)
+	{ visibilityView.visible_(false); };
+});
+
+// Same for the text view :
+textHideButton.action_({
+	textView.visible_(false);
+	textShowButton.visible_(true);
+	numberOfHiddenView = numberOfHiddenView + 1;
+	if(numberOfHiddenView == 1)
+	{ visibilityView.visible_(true); };
+});
+
+textShowButton.action_({
+	textView.visible_(true);
+	textShowButton.visible_(false);
+	numberOfHiddenView = numberOfHiddenView - 1;
+	if(numberOfHiddenView == 0)
+	{ visibilityView.visible_(false); };
+});
+
+// Done
+window.front;
+)
+::
```

Comment:
I doubt this is the last time I forget an empty line as EOF :) .

"Things you wanna know" is more an appendix to the tutorial, rather than a chapter. In fact, the reason for it is almost only to be able to definitely close #3449 in a 'proper way'. But this also allows for an official disclaimer against QT related issues (which arise from times to times on the forum, or on issues here), and also leaves a space for anyone to document Qt related stuff (for example, #3755, multi-touch currently unsupported, could be mentioned on this file).